### ISSUE TITLE
Turnkey Owncloud VM - stuck on partitioning

### ISSUE THREAD

**Issue by mygitus** [community-scripts/ProxmoxVE#93]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

Hi!
I am trying to install the Turnkey Owncloud VM using the default settings and choosing local-lvm for storage. I end up with a VM with a disc size of 692M (shown under the VM's "Hardware" in Proxmox). When starting the VM I get to the "Debian Installer Live". There I am asked too choose "_Partitioning method_".  When I choose "_Guided - use entire disk and set LVM_", I get this error message:
"_Failed to partition the selected disk
This probably happened because the selected disc or free space is too small to be automatically partitioned_"

Thanks for the help and for many great scripts!

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### Please provide detailed steps to reproduce the issue.

1. Run the script: bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/vm/owncloud-vm.sh)"
2. choose local-lvm for storage
3. Start the VM

**Comment by havardthom** [comment]
The disk with size 692M is the bootdisk. You need to add another disk for storage to partition.

![add-disk](https://github.com/user-attachments/assets/33b8f465-10c2-4c0f-9b6e-c34de45ee46d)
![partition](https://github.com/user-attachments/assets/236081b3-efe6-405c-abaf-be98fdb679ba)

**Comment by havardthom** [comment]
Can probably add a note about this in docs

**Comment by mygitus** [comment]
Yes of course... I should have understood that. I guess the info on the webpage (https://community-scripts.github.io/Proxmox/scripts?id=ownCloud), made me expect to see a 8GB disk created by the script.
<img width="260" alt="Screenshot owncloud" src="https://github.com/user-attachments/assets/92eedce7-5ece-4ef9-98d0-961ddb256dca">

Thanks!

**Comment by havardthom** [comment]
hm the script should actually set up a disk, reopening

**Comment by Mellowlynx** [comment]
> hm the script should actually set up a disk, reopening

That's what I thought, so I did set the right label  😃
